lldb/source/Plugins/Language/Swift/SwiftOptional.cpp

```diff
@@ -86,10 +93,10 @@ ExtractSomeIfAny(ValueObject *optional,
           DataExtractor extractor(buffer_sp, process_sp->GetByteOrder(),
                                   process_sp->GetAddressByteSize());
           ExecutionContext exe_ctx(process_sp);
-          value_sp = PointerOrSP(ValueObject::CreateValueObjectFromData(
+          value_sp = ValueObjectSP(ValueObject::CreateValueObjectFromData(
```

Comment:
`CreateValueObjectFromData` returns a `ValueObjectSP` so the explicit `ValueObjectSP(...)` construction isn't needed.